org.hl7.fhir.dstu3/src/main/java/org/hl7/fhir/dstu3/utils/client/network/FhirRequestBuilder.java

```diff
@@ -302,6 +307,26 @@ protected IParser getParser(String format) {
     }
   }
 
+  /**
+   * Logs the given {@link Request}, using the current {@link ToolingClientLogger}. If the current
+   * {@link FhirRequestBuilder#logger} is null, no action is taken.
+   *
+   * @param method  HTTP request method
+   * @param url request URL
+   * @param requestHeaders {@link Headers} for request
+   * @param requestBody Byte array request
+   */
+  protected void log(String method, String url, Headers requestHeaders, byte[] requestBody) {
+    if (logger != null) {
+      List<String> headerList = new ArrayList<>(Collections.emptyList());
+      Map<String, List<String>> headerMap = requestHeaders.toMultimap();
+      headerMap.keySet().forEach(key -> headerMap.get(key).forEach(value -> headerList.add(key + ":" + value)));
+
+      logger.logRequest(method, url, headerList, requestBody);
+    }
+
+  }
+
```

Comment:
Are there any other places in the request builder that can use this method?

Reply:
There are only two references to getHTTPClient(), both of which have this log(...) method applied.

All other references to the okHttpClient are within the getHTTPClient method itself.

I don't think there's anywhere else this could be used.